src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -131,6 +131,9 @@ def load_arguments(self, _):
 
     with self.argument_context('appservice plan update') as c:
         c.argument('sku', arg_type=sku_arg_type)
+        c.argument('elastic_scale', arg_type=get_three_state_flag(), is_preview=True, help='Enable or disable automatic scaling. Set to "true" to enable elastic scale for this plan, or "false" to disable elastic scale for this plan. The SKU must be a Premium V2 SKU (P1V2, P2V2, P3V2) or a Premium V3 SKU (P1V3, P2V3, P3V3)')
```

Comment:
Could we consider using `get_three_state_flag()` instead of `action='store_true'` ?

Reply:
Since this is an update command the parameter needs to be `True` or `False` if the user sets a value for the parameter and `None` if they do not (we don't want elastic scale to be accidentally disabled when someone runs an update command without using `--elastic-scale`) Does using `action='store_true'` allow for the argument to be `None` as well as `True`/`False`?

Reply:
Got it, thanks~